src/lobject.h

```diff
@@ -569,6 +569,13 @@ typedef struct Proto {
   TValue *k;  /* constants used by the function */
   Instruction *code;  /* opcodes */
 #ifdef USE_YK
+  /* Used to detect recursive function calls. When a function is
```

Comment:
If I were to nitpick (moi?), I'd format the comment like the others (with a star on each line).